### PR TITLE
change specter-desktop to specter

### PR DESCRIPTION
**Pull request by BitcoinMechanic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,8 @@ cd embassy-os/backend/
 Clone the project locally. Note the submodule link to the original project(s). 
 
 ```
-git clone https://github.com/Start9Labs/specter-desktop-wrapper.git
-cd specter-desktop-wrapper
+git clone https://github.com/Start9Labs/specter-wrapper.git
+cd specter--wrapper
```
Comment (MattDHill) [inline README.md:67]: Double dash

Comment (BitcoinMechanic) [reply README.md]: Good catch - fixed

File: manifest.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+wrapper-repo: https://github.com/Start9Labs/specter-wrapper
 upstream-repo: https://github.com/cryptoadvance/specter-desktop
 support-site: https://github.com/cryptoadvance/specter-desktop/issues
 marketing-site: https://specter.solutions
 build: ["make"]
 description:
-  short: A desktop GUI for Bitcoin Core optimised to work with hardware wallets 
-  long: A desktop GUI for Bitcoin Core optimised to work with hardware wallets. This software might be ready to be used but at your own risk. 
+  short: A GUI for Bitcoin Core optimised to work with hardware wallets 
+  long: A GUI for Bitcoin Core optimised to work with hardware wallets. This software might be ready to be used but at your own risk.
```
[the remark this answers is not in the document]
Comment (BitcoinMechanic) [reply manifest.yaml:13]: Will do but in a new PR.

File: README.md [hunk 3]
```diff
@@ -84,10 +84,10 @@ Run the following command to install the package:
 ```
 embassy-cli auth login
 #Enter your embassy password then run:
-embassy-cli package install /path/to/specter-desktop.s9pk
+embassy-cli package install /path/to/specter.s9pk
 ```
 ## Verify Install
 
-Go to your Embassy Services page, select specter-desktop and start the service.
+Go to your Embassy Services page, select specter and start the service.
```
Comment (kn0wmad) [inline README.md:91]: Capitalize Specter

**Review by kn0wmad — APPROVED** [review]
One minor typo, otherwise lgtm

File: assets/compat/btc-rpc-proxy_config_rules.yaml [hunk 4]
```diff
@@ -1,332 +1,332 @@
-- rule: '''users.*.name = "specter-desktop"'
-  description: 'Must have an RPC user named "specter-desktop"'
+- rule: '''users.*.name = "specter"'
+  description: 'Must have an RPC user named "specter"'
```
Comment (chrisguida) [inline assets/compat/btc-rpc-proxy_config_rules.yaml:2]: This will conflict with external specter instances. Though perhaps that's fine

**Pull request merged**